west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: syst_upstream
```

Comment:
Hmm shouldn't we be mirroring this at the project level, instead of adding a new upstream?

Reply:
this is still WIP I guess :)

Reply:
@andrewboie @nashif put the changes for syst on my own fork first, when everything is good, will do the mirroring at the project level.